layouts/partials/pagination.html

```diff
@@ -1,6 +1,6 @@
 {{ if or ( .Paginator.HasPrev ) ( .Paginator.HasNext ) }}
-    {{ $toNewPostsMessage := i18n "toNewPosts" | humanize }}
-    {{ $toOldPostsMessage := i18n "toOldPosts" | humanize }}    
+    {{ $toNewPostsMessage := i18n "toNewPosts" }}
```

Comment:
Ok, I read comment to your commit and understand your point.  
But it would be better to keep first letter capitalization on all translations by default for this case.  
Could you please update translation files for this?